components/Button/RandomButton.styled.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { Merriweather_Sans } from "next/font/google";
+
+const merriweatherSans = Merriweather_Sans({ subsets: ["latin"] });
+
+export const StyledRandomButton = styled.button`
+  font-family: ${merriweatherSans.style.fontFamily};
+  display: inline-flex;
+  align-items: center;
+  color: white;
+  border: 0.2rem solid green;
+  background: linear-gradient(
+    270deg,
+    rgba(0, 150, 64, 1) 13%,
+    rgba(47, 172, 102, 1) 29%,
+    rgba(0, 102, 51, 1) 58%
+  );
+  font-size: small;
+  padding: 0.5rem 0.5rem;
+  aspect-ratio: 1;
+  border-radius: 8rem;
+  box-shadow: 0px 1px 5px -2px green;
+  margin: 1rem;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &:visited {
+    color: white;
+  }
```

Comment:
The pseudoclass `:visited`  only applies on links.

Reply:
ok thx